shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/App.kt

```diff
@@ -36,78 +37,109 @@ import org.jetbrains.kotlinconf.screens.Speaker
-                    navController = navController,
-                    startDestination = StartScreen,
-                    modifier = Modifier.fillMaxSize(),
-                    enterTransition = enterTransition { it },
-                    exitTransition = exitTransition { -it },
-                    popEnterTransition = enterTransition { -it },
-                    popExitTransition = exitTransition { it },
+    DevelopmentEntryPoint {
+        KotlinConfTheme {
+            val service = remember {
+                ConferenceService(context, apiEndpoint)
+            }
+            CompositionLocalProvider(LocalNavController provides navController) {
+                Box(
+                    Modifier.fillMaxSize()
+                        .background(KotlinConfTheme.colors.mainBackground),
+                    contentAlignment = Alignment.Center
                 ) {
-                    composable<StartScreen> {
-                        Column {
-                            StyledText("Start screen")
-                            StyledText("About App", Modifier.clickable { navController.navigate(AboutAppScreen) })
-                            StyledText("About Conference", Modifier.clickable { navController.navigate(AboutConferenceScreen) })
-                            StyledText("Code of Conduct", Modifier.clickable { navController.navigate(CodeOfConductScreen) })
-                            StyledText("Settings", Modifier.clickable { navController.navigate(SettingsScreen) })
-                            StyledText("Terms of use", Modifier.clickable { navController.navigate(TermsOfUseScreen) })
-                            StyledText("Partners", Modifier.clickable { navController.navigate(PartnersScreen) })
-                            StyledText("Schedule", Modifier.clickable { navController.navigate(ScheduleScreen) })
-                            StyledText("Speakers", Modifier.clickable { navController.navigate(SpeakersScreen) })
+                    NavHost(
+                        navController = navController,
+                        startDestination = StartScreen,
+                        modifier = Modifier.fillMaxSize(),
+                        enterTransition = enterTransition { it },
+                        exitTransition = exitTransition { -it },
+                        popEnterTransition = enterTransition { -it },
+                        popExitTransition = exitTransition { it },
+                    ) {
+                        composable<StartScreen> {
+                            Column {
+                                StyledText("Start screen")
+                                StyledText(
+                                    "Privacy policy",
+                                    Modifier.clickable { navController.navigate(StartPrivacyPolicyScreen) }
+                                )
+                                StyledText("About App", Modifier.clickable { navController.navigate(AboutAppScreen) })
```

Comment:
Please consider extracting string constant to resources

Reply:
All of this in `App.kt` is placeholder buttons at the moment, just to be able to open screens until we have the main nav built out, so it'll be removed anyway